roles/translator/src/proxy/bridge.rs

```diff
@@ -270,48 +563,70 @@ impl Bridge {
                     "handle_new_prev_hash job_id: {:?}",
                     &sv2_set_new_prev_hash.job_id
                 );
+                Self::handle_new_prev_hash_(
+                    self_.clone(),
+                    sv2_set_new_prev_hash,
+                    tx_status.clone(),
+                    tx_sv1_notify.clone(),
+                )
+                .await;
+            }
+        });
+    }
+
+    async fn handle_new_extended_mining_job_(
+        self_: Arc<Mutex<Self>>,
+        sv2_new_extended_mining_job: &NewExtendedMiningJob<'static>,
+        tx_status: status::Sender,
+        tx_sv1_notify: broadcast::Sender<server_to_client::Notify<'static>>,
+    ) {
+        loop {
```

Comment:
I'm confused as to why this is a loop{} that only goes through the block once and all cases end with a `break`